src/processor.cpp

```diff
@@ -0,0 +1,164 @@
+// This file is a part of Julia. License is MIT: https://julialang.org/license
+
+// Processor feature detection
+
+#include "julia.h"
+#include "julia_internal.h"
+
+extern "C" {
```

Comment:
LLVM does have it but it's not exposing all the details (e.g. I find no way to access what features is enabled by a cpu, which may or may not be needed). More importantly we should be able to load the multi-versioning sysimg with LLVM disabled (or we need a `antiprocessor.c`............)

OpenBLAS also has it, though obviously not exposed in a useful way.

I'm not aware of such code in libuv.....